lib/modules/manager/cpanfile/index.ts

```diff
@@ -0,0 +1,18 @@
+import type { ProgrammingLanguage } from '../../../constants';
+import { CpanDatasource } from '../../datasource/cpan';
+import * as perlVersioning from '../../versioning/perl';
+
+export { extractPackageFile } from './extract';
+
+export const displayName = 'cpanfile';
+export const url =
+  'https://metacpan.org/dist/Module-CPANfile/view/lib/cpanfile.pod';
+
+export const language: ProgrammingLanguage = 'perl';
+
+export const defaultConfig = {
+  fileMatch: ['(^|/)cpanfile$'],
+  versioning: perlVersioning.id,
```

Comment:
```suggestion
```
should be set on cpan datasource default versioning or on perl dep for tags datasource